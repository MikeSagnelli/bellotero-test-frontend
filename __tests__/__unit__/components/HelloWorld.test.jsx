import React from 'react';
import { shallow } from 'enzyme';
import HelloWorld from '../../../components/HelloWorld';

describe('Test Hello World', () => {
  it('renders hello world header', () => {
    const component = shallow(<HelloWorld />);
    expect(component).toMatchSnapshot();
  });
});
