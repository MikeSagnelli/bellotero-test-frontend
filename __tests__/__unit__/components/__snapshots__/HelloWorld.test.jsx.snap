// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Hello World renders hello world header 1`] = `
<h1>
  Hello World!
</h1>
`;
